packages/types/docs/modules.md

```diff
@@ -161,47 +161,47 @@
 
 ### Account
 
-Ƭ **Account**: `Pick`\<[`DerivedAccount`](modules.md#derivedaccount), ``"address"`` \| ``"alias"`` \| ``"type"`` \| ``"publicKey"`` \| ``"owner"`` \| ``"pseudoExtendedKey"`` \| ``"source"`` \| ``"timestamp"``\> & \{ `viewingKey?`: `string`  }
+Ƭ **Account**: `Pick`\<[`DerivedAccount`](modules.md#derivedaccount), `"address"` \| `"alias"` \| `"type"` \| `"publicKey"` \| `"owner"` \| `"pseudoExtendedKey"` \| `"source"` \| `"timestamp"`\> & \{ `viewingKey?`: `string` }
```

Comment:
Question: how was this file modified? We normally update docs via `yarn build:docs` in the `types` & `sdk` package to automatically pick up changes, and don't manually edit these. Though we also generally do this prior to a release if there are any changes! I think it's safe to revert this change, and we can later regen the type docs here!